### ISSUE TITLE
Run code in documentation regularly, to prevent breaks

### ISSUE THREAD

**Issue by yaeldekel** [dotnet/machinelearning#2055]
We need to figure out a way to run the code in the documentation, perhaps using a unit test.
Related to #2039 .
@justinormont , in the issue above it is mentioned that you had already suggested some sort of solution to this, please feel free to reference the issue discussing this, or close this issue if it is a duplicate.

**Comment by justinormont** [comment]
I don't think we previously filed an issue about ensuring our example code stays running (perhaps wrapping in a unit test). Hence, this issues isn't a duplicate.

**Comment by artidoro** [comment]
We should consider running the samples as part of our CI.
However, we ensure that the code compiles since our samples are part of the main repo.
